openssh/files/sshd_config

```diff
@@ -50,6 +50,10 @@ AuthorizedKeysCommand {{ server.authorized_keys_command.command }}
 AuthorizedKeysCommandUser {{ server.authorized_keys_command.get('user', 'nobody') }}
 {%- endif %}
 
+{% if server.get('deny_users', False) %}
```

Comment:
You should use list and then join it with space separator.
The same can be done for AllowUsers, DenyGroups, AllowGroups